[package] spdlog/1.7.0: header only option is inverted

<!-- 
  Please don't forget to update the issue title.
  Include all applicable information to help us reproduce your problem.
-->

## Description

Since 20.07.2020 I noticed that our gitlab CI/CD runner could not build our internal projects and I narrowed it down to our logging frontend failling to be built. I am using CMake integration module to update conan depenednencies at configuration time, I provided a full log bellow, however I can not provide the project itself at this moment. 

From what I can see the problem is caused by using the prebuilt **fmt** library when `header-only:true`:
https://github.com/conan-io/conan-center-index/blob/b1d8a1d4077d68d94e0f07b604d58504f8476799/recipes/spdlog/all/conanfile.py#L121

@prince-chrismc introduced this change in 684a283

Could this be a mix up in that commit or is it intended to be like that?

### Package and Environment Details (include every applicable attribute)
  * Package Name/Version: **spdlog/1.7.0**
  * Operating System+version: **"Debian GNU/Linux 10**
  * Compiler+version: **GCC 8.3.0**
  * Docker image: **none**
  * Conan version: **conan 1.27.1**
  * Python version: **Python 3.7.3**


### Conan profile (output of `conan profile show default` or `conan profile show <profile>` if custom profile is in use)
```
Configuration for profile default:

[settings]
os=Linux
os_build=Linux
arch=x86_64
arch_build=x86_64
compiler=gcc
compiler.version=8
compiler.libcxx=libstdc++11
build_type=Release
[options]
[build_requires]
[env]

```

### Logs (Include/Attach if Applicable)
<details><summary>Click to expand log</summary>

```
-- The CXX compiler identification is GNU 8.3.0
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Using GNU compiler
-- Found program: /home/gitlab-runner/.local/bin/conan
-- Using Conan Version Conan version 1.27.1

-- Using local conanfile: /home/gitlab-runner/hasll/conanfile.txt
Configuration:
[settings]
arch=x86_64
arch_build=x86_64
build_type=Release
compiler=gcc
compiler.libcxx=libstdc++11
compiler.version=8
os=Linux
os_build=Linux
[options]
[build_requires]
[env]

conanfile.txt: Installing package
Requirements
    fmt/7.0.1 from 'conan-center' - Cache
    gtest/1.8.1@bincrafters/stable from 'conan-center' - Cache
    nlohmann_json/3.7.3 from 'conan-center' - Cache
    spdlog/1.7.0 from 'conan-center' - Cache
Packages
    fmt/7.0.1:56e0cf6d16ee57367a0661ab743f4e43b29223f8 - Cache
    gtest/1.8.1@bincrafters/stable:319986a8ddc0fc392032c63f81cacfe9edd93e70 - Cache
    nlohmann_json/3.7.3:5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9 - Cache
    spdlog/1.7.0:c7e1b1004c948250b7562beecd9031d66e8b92a7 - Cache

Installing (downloading, building) binaries...
fmt/7.0.1: Already installed!
gtest/1.8.1@bincrafters/stable: Already installed!
nlohmann_json/3.7.3: Already installed!
spdlog/1.7.0: Already installed!
conanfile.txt: Generator cmake_paths created conan_paths.cmake

CMake Warning at cmake/conan_install.cmake:51 (message):
  Conan failed to install failed!
Call Stack (most recent call first):
  CMakeLists.txt:20 (execute_conan_install)


conanfile.txt: ERROR: Generator cmake_find_package(file:None) failed
Component 'fmt::fmt-header-only' not found in 'fmt' package requirement
ERROR: Component 'fmt::fmt-header-only' not found in 'fmt' package requirement

CMake Warning at cmake/conan_install.cmake:53 (message):
  Trying to build sources locally!
Call Stack (most recent call first):
  CMakeLists.txt:20 (execute_conan_install)


Configuration:
[settings]
arch=x86_64
arch_build=x86_64
build_type=Release
compiler=gcc
compiler.libcxx=libstdc++11
compiler.version=8
os=Linux
os_build=Linux
[options]
[build_requires]
[env]

fmt/7.0.1: Forced build from source
gtest/1.8.1@bincrafters/stable: Forced build from source
nlohmann_json/3.7.3: Forced build from source
spdlog/1.7.0: Forced build from source
conanfile.txt: Installing package
Requirements
    fmt/7.0.1 from 'conan-center' - Cache
    gtest/1.8.1@bincrafters/stable from 'conan-center' - Cache
    nlohmann_json/3.7.3 from 'conan-center' - Cache
    spdlog/1.7.0 from 'conan-center' - Cache
Packages
    fmt/7.0.1:56e0cf6d16ee57367a0661ab743f4e43b29223f8 - Build
    gtest/1.8.1@bincrafters/stable:319986a8ddc0fc392032c63f81cacfe9edd93e70 - Build
    nlohmann_json/3.7.3:5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9 - Build
    spdlog/1.7.0:c7e1b1004c948250b7562beecd9031d66e8b92a7 - Build

Installing (downloading, building) binaries...
fmt/7.0.1: Copying sources to build folder
fmt/7.0.1: Building your package in /home/gitlab-runner/.conan/data/fmt/7.0.1/_/_/build/56e0cf6d16ee57367a0661ab743f4e43b29223f8
fmt/7.0.1: Generator cmake created conanbuildinfo.cmake
fmt/7.0.1: Calling build()
-- The C compiler identification is GNU 8.3.0
-- The CXX compiler identification is GNU 8.3.0
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Conan: called by CMake conan helper
-- Conan: called inside local cache
-- Conan: Adjusting output directories
-- Conan: Using cmake global configuration
-- Conan: Adjusting default RPATHs Conan policies
-- Conan: Adjusting language standard
-- Conan: Adjusting fPIC flag (ON)
-- Conan: Compiler GCC>=5, checking major version 8
-- Conan: Checking correct version: 8
-- Conan: C++ stdlib: libstdc++11
-- Version: 7.0.1
-- Build type: Release
-- CXX_STANDARD: 11
-- Performing Test has_std_11_flag
-- Performing Test has_std_11_flag - Success
-- Performing Test has_std_0x_flag
-- Performing Test has_std_0x_flag - Success
-- Performing Test SUPPORTS_USER_DEFINED_LITERALS
-- Performing Test SUPPORTS_USER_DEFINED_LITERALS - Success
-- Performing Test FMT_HAS_VARIANT
-- Performing Test FMT_HAS_VARIANT - Success
-- Required features: cxx_variadic_templates
-- Looking for strtod_l
-- Looking for strtod_l - not found
-- Configuring done
-- Generating done
CMake Warning:
  Manually-specified variables were not used by the project:

    CMAKE_EXPORT_NO_PACKAGE_REGISTRY


-- Build files have been written to: /home/gitlab-runner/.conan/data/fmt/7.0.1/_/_/build/56e0cf6d16ee57367a0661ab743f4e43b29223f8/build_subfolder
Scanning dependencies of target fmt
[ 66%] Building CXX object source_subfolder/CMakeFiles/fmt.dir/src/format.cc.o
[ 66%] Building CXX object source_subfolder/CMakeFiles/fmt.dir/src/os.cc.o
[100%] Linking CXX static library ../lib/libfmt.a
[100%] Built target fmt
fmt/7.0.1: Package '56e0cf6d16ee57367a0661ab743f4e43b29223f8' built
fmt/7.0.1: Build folder /home/gitlab-runner/.conan/data/fmt/7.0.1/_/_/build/56e0cf6d16ee57367a0661ab743f4e43b29223f8
fmt/7.0.1: Generated conaninfo.txt
fmt/7.0.1: Generated conanbuildinfo.txt
fmt/7.0.1: Generating the package
fmt/7.0.1: Package folder /home/gitlab-runner/.conan/data/fmt/7.0.1/_/_/package/56e0cf6d16ee57367a0661ab743f4e43b29223f8
fmt/7.0.1: Calling package()
[100%] Built target fmt
Install the project...
-- Install configuration: "Release"
-- Installing: /home/gitlab-runner/.conan/data/fmt/7.0.1/_/_/package/56e0cf6d16ee57367a0661ab743f4e43b29223f8/lib/cmake/fmt/fmt-config.cmake
-- Installing: /home/gitlab-runner/.conan/data/fmt/7.0.1/_/_/package/56e0cf6d16ee57367a0661ab743f4e43b29223f8/lib/cmake/fmt/fmt-config-version.cmake
-- Installing: /home/gitlab-runner/.conan/data/fmt/7.0.1/_/_/package/56e0cf6d16ee57367a0661ab743f4e43b29223f8/lib/cmake/fmt/fmt-targets.cmake
-- Installing: /home/gitlab-runner/.conan/data/fmt/7.0.1/_/_/package/56e0cf6d16ee57367a0661ab743f4e43b29223f8/lib/cmake/fmt/fmt-targets-release.cmake
-- Installing: /home/gitlab-runner/.conan/data/fmt/7.0.1/_/_/package/56e0cf6d16ee57367a0661ab743f4e43b29223f8/lib/libfmt.a
-- Installing: /home/gitlab-runner/.conan/data/fmt/7.0.1/_/_/package/56e0cf6d16ee57367a0661ab743f4e43b29223f8/include/fmt/chrono.h
-- Installing: /home/gitlab-runner/.conan/data/fmt/7.0.1/_/_/package/56e0cf6d16ee57367a0661ab743f4e43b29223f8/include/fmt/color.h
-- Installing: /home/gitlab-runner/.conan/data/fmt/7.0.1/_/_/package/56e0cf6d16ee57367a0661ab743f4e43b29223f8/include/fmt/compile.h
-- Installing: /home/gitlab-runner/.conan/data/fmt/7.0.1/_/_/package/56e0cf6d16ee57367a0661ab743f4e43b29223f8/include/fmt/core.h
-- Installing: /home/gitlab-runner/.conan/data/fmt/7.0.1/_/_/package/56e0cf6d16ee57367a0661ab743f4e43b29223f8/include/fmt/format.h
-- Installing: /home/gitlab-runner/.conan/data/fmt/7.0.1/_/_/package/56e0cf6d16ee57367a0661ab743f4e43b29223f8/include/fmt/format-inl.h
-- Installing: /home/gitlab-runner/.conan/data/fmt/7.0.1/_/_/package/56e0cf6d16ee57367a0661ab743f4e43b29223f8/include/fmt/locale.h
-- Installing: /home/gitlab-runner/.conan/data/fmt/7.0.1/_/_/package/56e0cf6d16ee57367a0661ab743f4e43b29223f8/include/fmt/os.h
-- Installing: /home/gitlab-runner/.conan/data/fmt/7.0.1/_/_/package/56e0cf6d16ee57367a0661ab743f4e43b29223f8/include/fmt/ostream.h
-- Installing: /home/gitlab-runner/.conan/data/fmt/7.0.1/_/_/package/56e0cf6d16ee57367a0661ab743f4e43b29223f8/include/fmt/posix.h
-- Installing: /home/gitlab-runner/.conan/data/fmt/7.0.1/_/_/package/56e0cf6d16ee57367a0661ab743f4e43b29223f8/include/fmt/printf.h
-- Installing: /home/gitlab-runner/.conan/data/fmt/7.0.1/_/_/package/56e0cf6d16ee57367a0661ab743f4e43b29223f8/include/fmt/ranges.h
-- Installing: /home/gitlab-runner/.conan/data/fmt/7.0.1/_/_/package/56e0cf6d16ee57367a0661ab743f4e43b29223f8/lib/pkgconfig/fmt.pc
fmt/7.0.1 package(): Packaged 1 '.a' file: libfmt.a
fmt/7.0.1 package(): Packaged 1 '.rst' file: LICENSE.rst
fmt/7.0.1 package(): Packaged 12 '.h' files
fmt/7.0.1: Package '56e0cf6d16ee57367a0661ab743f4e43b29223f8' created
fmt/7.0.1: Created package revision 8e8080e29c6bc9963d0132519b1cba1d
gtest/1.8.1@bincrafters/stable: Copying sources to build folder
gtest/1.8.1@bincrafters/stable: Building your package in /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/build/319986a8ddc0fc392032c63f81cacfe9edd93e70
gtest/1.8.1@bincrafters/stable: Generator cmake created conanbuildinfo.cmake
gtest/1.8.1@bincrafters/stable: Calling build()
-- The C compiler identification is GNU 8.3.0
-- The CXX compiler identification is GNU 8.3.0
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Conan: called by CMake conan helper
-- Conan: called inside local cache
-- Conan: Adjusting output directories
-- Conan: Using cmake global configuration
-- Conan: Adjusting default RPATHs Conan policies
-- Conan: Adjusting language standard
-- Conan: Adjusting fPIC flag (ON)
-- Conan: Compiler GCC>=5, checking major version 8
-- Conan: Checking correct version: 8
-- Conan: C++ stdlib: libstdc++11
-- Found PythonInterp: /usr/bin/python3.7 (found version "3.7.3")
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Failed
-- Check if compiler accepts -pthread
-- Check if compiler accepts -pthread - yes
-- Found Threads: TRUE
-- Configuring done
-- Generating done
CMake Warning:
  Manually-specified variables were not used by the project:

    CMAKE_EXPORT_NO_PACKAGE_REGISTRY


-- Build files have been written to: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/build/319986a8ddc0fc392032c63f81cacfe9edd93e70
Scanning dependencies of target gtest
[ 12%] Building CXX object source_subfolder/googlemock/gtest/CMakeFiles/gtest.dir/src/gtest-all.cc.o
[ 25%] Linking CXX static library ../../../lib/libgtest.a
[ 25%] Built target gtest
Scanning dependencies of target gtest_main
[ 37%] Building CXX object source_subfolder/googlemock/gtest/CMakeFiles/gtest_main.dir/src/gtest_main.cc.o
Scanning dependencies of target gmock
[ 50%] Building CXX object source_subfolder/googlemock/CMakeFiles/gmock.dir/src/gmock-all.cc.o
[ 62%] Linking CXX static library ../../../lib/libgtest_main.a
[ 62%] Built target gtest_main
[ 75%] Linking CXX static library ../../lib/libgmock.a
[ 75%] Built target gmock
Scanning dependencies of target gmock_main
[ 87%] Building CXX object source_subfolder/googlemock/CMakeFiles/gmock_main.dir/src/gmock_main.cc.o
[100%] Linking CXX static library ../../lib/libgmock_main.a
[100%] Built target gmock_main
gtest/1.8.1@bincrafters/stable: Package '319986a8ddc0fc392032c63f81cacfe9edd93e70' built
gtest/1.8.1@bincrafters/stable: Build folder /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/build/319986a8ddc0fc392032c63f81cacfe9edd93e70
gtest/1.8.1@bincrafters/stable: Generated conaninfo.txt
gtest/1.8.1@bincrafters/stable: Generated conanbuildinfo.txt
gtest/1.8.1@bincrafters/stable: Generating the package
gtest/1.8.1@bincrafters/stable: Package folder /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70
gtest/1.8.1@bincrafters/stable: Calling package()
-- Conan: called by CMake conan helper
-- Conan: called inside local cache
-- Conan: Adjusting output directories
-- Conan: Using cmake global configuration
-- Conan: Adjusting default RPATHs Conan policies
-- Conan: Adjusting language standard
-- Conan: Adjusting fPIC flag (ON)
-- Conan: Compiler GCC>=5, checking major version 8
-- Conan: Checking correct version: 8
-- Conan: C++ stdlib: libstdc++11
-- Configuring done
-- Generating done
-- Build files have been written to: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/build/319986a8ddc0fc392032c63f81cacfe9edd93e70
[ 25%] Built target gtest
[ 50%] Built target gtest_main
[ 75%] Built target gmock
[100%] Built target gmock_main
Install the project...
-- Install configuration: "Release"
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/FindGTest.cmake
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/FindGMock.cmake
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include/gmock
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include/gmock/gmock-generated-matchers.h
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include/gmock/gmock-generated-actions.h.pump
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include/gmock/gmock-spec-builders.h
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include/gmock/gmock-more-actions.h
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include/gmock/gmock-generated-actions.h
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include/gmock/gmock.h
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include/gmock/gmock-more-matchers.h
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include/gmock/gmock-cardinalities.h
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include/gmock/gmock-matchers.h
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include/gmock/gmock-generated-function-mockers.h
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include/gmock/gmock-actions.h
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include/gmock/internal
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include/gmock/internal/gmock-internal-utils.h
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include/gmock/internal/gmock-generated-internal-utils.h
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include/gmock/internal/gmock-port.h
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include/gmock/internal/custom
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include/gmock/internal/custom/gmock-generated-actions.h.pump
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include/gmock/internal/custom/gmock-port.h
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include/gmock/internal/custom/gmock-generated-actions.h
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include/gmock/internal/custom/gmock-matchers.h
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include/gmock/internal/custom/README.md
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include/gmock/internal/gmock-generated-internal-utils.h.pump
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include/gmock/gmock-generated-matchers.h.pump
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include/gmock/gmock-generated-nice-strict.h
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include/gmock/gmock-generated-nice-strict.h.pump
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include/gmock/gmock-generated-function-mockers.h.pump
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/lib/libgmock.a
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/lib/libgmock_main.a
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/lib/pkgconfig/gmock.pc
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/lib/pkgconfig/gmock_main.pc
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/lib/cmake/GTest/GTestTargets.cmake
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/lib/cmake/GTest/GTestTargets-release.cmake
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/lib/cmake/GTest/GTestConfigVersion.cmake
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/lib/cmake/GTest/GTestConfig.cmake
-- Up-to-date: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include/gtest
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include/gtest/gtest-test-part.h
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include/gtest/gtest-message.h
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include/gtest/gtest-death-test.h
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include/gtest/gtest-spi.h
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include/gtest/gtest_prod.h
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include/gtest/gtest_pred_impl.h
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include/gtest/gtest-param-test.h.pump
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include/gtest/gtest-printers.h
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include/gtest/internal
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include/gtest/internal/gtest-type-util.h.pump
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include/gtest/internal/gtest-port.h
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include/gtest/internal/gtest-param-util-generated.h
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include/gtest/internal/gtest-linked_ptr.h
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include/gtest/internal/gtest-param-util-generated.h.pump
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include/gtest/internal/gtest-port-arch.h
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include/gtest/internal/gtest-filepath.h
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include/gtest/internal/gtest-param-util.h
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include/gtest/internal/custom
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include/gtest/internal/custom/gtest-port.h
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include/gtest/internal/custom/README.md
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include/gtest/internal/custom/gtest-printers.h
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include/gtest/internal/custom/gtest.h
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include/gtest/internal/gtest-tuple.h.pump
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include/gtest/internal/gtest-internal.h
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include/gtest/internal/gtest-death-test-internal.h
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include/gtest/internal/gtest-string.h
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include/gtest/internal/gtest-tuple.h
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include/gtest/internal/gtest-type-util.h
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include/gtest/gtest-typed-test.h
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include/gtest/gtest.h
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/include/gtest/gtest-param-test.h
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/lib/libgtest.a
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/lib/libgtest_main.a
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/lib/pkgconfig/gtest.pc
-- Installing: /home/gitlab-runner/.conan/data/gtest/1.8.1/bincrafters/stable/package/319986a8ddc0fc392032c63f81cacfe9edd93e70/lib/pkgconfig/gtest_main.pc
gtest/1.8.1@bincrafters/stable package(): Packaged 6 '.cmake' files
gtest/1.8.1@bincrafters/stable package(): Packaged 4 '.a' files: libgtest_main.a, libgmock_main.a, libgtest.a, libgmock.a
gtest/1.8.1@bincrafters/stable package(): Packaged 4 '.pc' files: gmock.pc, gmock_main.pc, gtest_main.pc, gtest.pc
gtest/1.8.1@bincrafters/stable package(): Packaged 1 file: LICENSE
gtest/1.8.1@bincrafters/stable package(): Packaged 41 '.h' files
gtest/1.8.1@bincrafters/stable package(): Packaged 10 '.pump' files
gtest/1.8.1@bincrafters/stable package(): Packaged 2 '.md' files: README.md, README.md
gtest/1.8.1@bincrafters/stable: Package '319986a8ddc0fc392032c63f81cacfe9edd93e70' created
gtest/1.8.1@bincrafters/stable: Created package revision a4e09b60580254326f4e6cdd2a521b5c
nlohmann_json/3.7.3: Building your package in /home/gitlab-runner/.conan/data/nlohmann_json/3.7.3/_/_/build/5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9
nlohmann_json/3.7.3: Generator txt created conanbuildinfo.txt
nlohmann_json/3.7.3: Calling build()
-- The CXX compiler identification is GNU 8.3.0
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Using the single-header code from /home/gitlab-runner/.conan/data/nlohmann_json/3.7.3/_/_/source/source_subfolder/single_include/
-- Configuring done
-- Generating done
CMake Warning:
  Manually-specified variables were not used by the project:

    CMAKE_EXPORT_NO_PACKAGE_REGISTRY
    CONAN_COMPILER
    CONAN_COMPILER_VERSION
    CONAN_CXX_FLAGS
    CONAN_C_FLAGS
    CONAN_EXPORTED
    CONAN_IN_LOCAL_CACHE
    CONAN_LIBCXX
    CONAN_SHARED_LINKER_FLAGS


-- Build files have been written to: /home/gitlab-runner/.conan/data/nlohmann_json/3.7.3/_/_/build/5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9
nlohmann_json/3.7.3: Package '5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9' built
nlohmann_json/3.7.3: Build folder /home/gitlab-runner/.conan/data/nlohmann_json/3.7.3/_/_/build/5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9
nlohmann_json/3.7.3: Generated conaninfo.txt
nlohmann_json/3.7.3: Generated conanbuildinfo.txt
nlohmann_json/3.7.3: Generating the package
nlohmann_json/3.7.3: Package folder /home/gitlab-runner/.conan/data/nlohmann_json/3.7.3/_/_/package/5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9
nlohmann_json/3.7.3: Calling package()
-- Using the single-header code from /home/gitlab-runner/.conan/data/nlohmann_json/3.7.3/_/_/source/source_subfolder/single_include/
-- Configuring done
-- Generating done
-- Build files have been written to: /home/gitlab-runner/.conan/data/nlohmann_json/3.7.3/_/_/build/5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9
Install the project...
-- Install configuration: "Release"
-- Installing: /home/gitlab-runner/.conan/data/nlohmann_json/3.7.3/_/_/package/5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9/include
-- Installing: /home/gitlab-runner/.conan/data/nlohmann_json/3.7.3/_/_/package/5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9/include/nlohmann
-- Installing: /home/gitlab-runner/.conan/data/nlohmann_json/3.7.3/_/_/package/5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9/include/nlohmann/json.hpp
-- Installing: /home/gitlab-runner/.conan/data/nlohmann_json/3.7.3/_/_/package/5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9/lib/cmake/nlohmann_json/nlohmann_jsonConfig.cmake
-- Installing: /home/gitlab-runner/.conan/data/nlohmann_json/3.7.3/_/_/package/5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9/lib/cmake/nlohmann_json/nlohmann_jsonConfigVersion.cmake
-- Installing: /home/gitlab-runner/.conan/data/nlohmann_json/3.7.3/_/_/package/5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9/lib/cmake/nlohmann_json/nlohmann_jsonTargets.cmake
nlohmann_json/3.7.3 package(): Packaged 1 '.MIT' file: LICENSE.MIT
nlohmann_json/3.7.3 package(): Packaged 1 '.hpp' file: json.hpp
nlohmann_json/3.7.3: Package '5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9' created
nlohmann_json/3.7.3: Created package revision 4eb7d1c05f699a75e700fffb6d806cc3
spdlog/1.7.0: Copying sources to build folder
spdlog/1.7.0: Building your package in /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/build/c7e1b1004c948250b7562beecd9031d66e8b92a7
spdlog/1.7.0: Generator cmake_find_package created Findfmt.cmake
spdlog/1.7.0: Generator cmake created conanbuildinfo.cmake
spdlog/1.7.0: Calling build()
-- The C compiler identification is GNU 8.3.0
-- The CXX compiler identification is GNU 8.3.0
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Conan: called by CMake conan helper
-- Conan: called inside local cache
-- Conan: Adjusting output directories
-- Conan: Using cmake global configuration
-- Conan: Adjusting default RPATHs Conan policies
-- Conan: Adjusting language standard
-- Conan: Adjusting fPIC flag (ON)
-- Conan: Compiler GCC>=5, checking major version 8
-- Conan: Checking correct version: 8
-- Conan: C++ stdlib: libstdc++11
-- Build spdlog: 1.7.0
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Failed
-- Looking for pthread_create in pthreads
-- Looking for pthread_create in pthreads - not found
-- Looking for pthread_create in pthread
-- Looking for pthread_create in pthread - found
-- Found Threads: TRUE
-- Build type: Release
-- Conan: Using autogenerated Findfmt.cmake
-- Found fmt: 7.0.1 (found suitable version "7.0.1", minimum required is "5.3.0")
-- Library fmt found /home/gitlab-runner/.conan/data/fmt/7.0.1/_/_/package/56e0cf6d16ee57367a0661ab743f4e43b29223f8/lib/libfmt.a
-- Found: /home/gitlab-runner/.conan/data/fmt/7.0.1/_/_/package/56e0cf6d16ee57367a0661ab743f4e43b29223f8/lib/libfmt.a
-- Generating install
-- Configuring done
-- Generating done
CMake Warning:
  Manually-specified variables were not used by the project:

    CMAKE_EXPORT_NO_PACKAGE_REGISTRY


-- Build files have been written to: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/build/c7e1b1004c948250b7562beecd9031d66e8b92a7
Scanning dependencies of target spdlog
[ 14%] Building CXX object source_subfolder/CMakeFiles/spdlog.dir/src/spdlog.cpp.o
[ 28%] Building CXX object source_subfolder/CMakeFiles/spdlog.dir/src/color_sinks.cpp.o
[ 42%] Building CXX object source_subfolder/CMakeFiles/spdlog.dir/src/stdout_sinks.cpp.o
[ 57%] Building CXX object source_subfolder/CMakeFiles/spdlog.dir/src/file_sinks.cpp.o
[ 71%] Building CXX object source_subfolder/CMakeFiles/spdlog.dir/src/async.cpp.o
[ 85%] Building CXX object source_subfolder/CMakeFiles/spdlog.dir/src/cfg.cpp.o
[100%] Linking CXX static library ../lib/libspdlog.a
[100%] Built target spdlog
spdlog/1.7.0: Package 'c7e1b1004c948250b7562beecd9031d66e8b92a7' built
spdlog/1.7.0: Build folder /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/build/c7e1b1004c948250b7562beecd9031d66e8b92a7
spdlog/1.7.0: Generated conaninfo.txt
spdlog/1.7.0: Generated conanbuildinfo.txt
spdlog/1.7.0: Generating the package
spdlog/1.7.0: Package folder /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7
spdlog/1.7.0: Calling package()
[100%] Built target spdlog
Install the project...
-- Install configuration: "Release"
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/pattern_formatter-inl.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/spdlog.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/common-inl.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/version.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/details
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/details/console_globals.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/details/log_msg_buffer.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/details/registry-inl.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/details/periodic_worker-inl.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/details/os.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/details/synchronous_factory.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/details/file_helper.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/details/os-inl.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/details/circular_q.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/details/log_msg.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/details/file_helper-inl.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/details/backtracer.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/details/registry.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/details/null_mutex.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/details/tcp_client-windows.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/details/fmt_helper.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/details/windows_include.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/details/backtracer-inl.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/details/mpmc_blocking_q.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/details/periodic_worker.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/details/log_msg-inl.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/details/log_msg_buffer-inl.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/details/tcp_client.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/details/thread_pool.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/details/thread_pool-inl.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/tweakme.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/common.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/async.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/fmt
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/fmt/fmt.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/fmt/bin_to_hex.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/fmt/ostr.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/async_logger-inl.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/spdlog-inl.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/logger.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/fwd.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/pattern_formatter.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/logger-inl.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/sinks
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/sinks/sink.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/sinks/ostream_sink.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/sinks/tcp_sink.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/sinks/ansicolor_sink-inl.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/sinks/stdout_color_sinks.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/sinks/systemd_sink.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/sinks/wincolor_sink.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/sinks/stdout_color_sinks-inl.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/sinks/win_eventlog_sink.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/sinks/daily_file_sink.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/sinks/stdout_sinks.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/sinks/syslog_sink.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/sinks/base_sink-inl.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/sinks/basic_file_sink-inl.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/sinks/dist_sink.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/sinks/rotating_file_sink.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/sinks/android_sink.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/sinks/ringbuffer_sink.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/sinks/wincolor_sink-inl.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/sinks/stdout_sinks-inl.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/sinks/basic_file_sink.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/sinks/ansicolor_sink.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/sinks/msvc_sink.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/sinks/base_sink.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/sinks/dup_filter_sink.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/sinks/null_sink.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/sinks/rotating_file_sink-inl.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/sinks/sink-inl.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/formatter.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/cfg
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/cfg/env.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/cfg/argv.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/cfg/log_levels.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/cfg/helpers-inl.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/cfg/helpers.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/include/spdlog/async_logger.h
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/lib/libspdlog.a
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/lib/pkgconfig/spdlog.pc
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/lib/cmake/spdlog/spdlogConfigTargets.cmake
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/lib/cmake/spdlog/spdlogConfigTargets-release.cmake
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/lib/cmake/spdlog/spdlogConfig.cmake
-- Installing: /home/gitlab-runner/.conan/data/spdlog/1.7.0/_/_/package/c7e1b1004c948250b7562beecd9031d66e8b92a7/lib/cmake/spdlog/spdlogConfigVersion.cmake
spdlog/1.7.0 package(): Packaged 1 '.a' file: libspdlog.a
spdlog/1.7.0 package(): Packaged 1 file: LICENSE
spdlog/1.7.0 package(): Packaged 76 '.h' files
spdlog/1.7.0: Package 'c7e1b1004c948250b7562beecd9031d66e8b92a7' created
spdlog/1.7.0: Created package revision 7f2e4f991eb83ed76de740ff4a97f8c6
conanfile.txt: Generator cmake_paths created conan_paths.cmake

CMake Error at cmake/conan_install.cmake:66 (message):
  conanfile.txt: ERROR: Generator cmake_find_package(file:None) failed

  Component 'fmt::fmt-header-only' not found in 'fmt' package requirement

  ERROR: Component 'fmt::fmt-header-only' not found in 'fmt' package
  requirement

Call Stack (most recent call first):
  CMakeLists.txt:20 (execute_conan_install)

```

</details>

--- a/recipes/spdlog/all/conanfile.py
+++ b/recipes/spdlog/all/conanfile.py
@@ -118,9 +118,9 @@ class SpdlogConan(ConanFile):
             
         self.cpp_info.components[component_name].defines.append("SPDLOG_FMT_EXTERNAL")
         if self.options["fmt"].header_only:
-            self.cpp_info.components[component_name].requires = ["fmt::fmt"]
-        else:
             self.cpp_info.components[component_name].requires = ["fmt::fmt-header-only"]
+        else:
+            self.cpp_info.components[component_name].requires = ["fmt::fmt"]
         if self.options.wchar_support:
             self.cpp_info.components[component_name].defines.append("SPDLOG_WCHAR_TO_UTF8_SUPPORT")
         if self.options.wchar_filenames:

